src/Jackalope/NodeType/NodeProcessor.php

```diff
@@ -178,13 +185,15 @@ private function processNodeWithType(Node $node, NodeType $nodeTypeDefinition)
                                 $value = $defaultValues;
                             } elseif (isset($defaultValues[0])) {
                                 $value = $defaultValues[0];
-                            } else {
+                            } elseif ($nodeTypeDefinition->getName() !== VersionHandler::MIX_VERSIONABLE) {
```

Comment:
I wonder if we could refactor this, something like:

````php
if ($propertyDef->isAutoCreated()) {
    $value = null;
    if ($nodeType != 'mix:versionable') {
        $value = $this->getDefaultValue($propertyDef);
    }
}
````

Otherwise is `$value` undefined if the node is MIX_VERSIONABLE?